logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/ClearUserDataUseCase.kt

```diff
@@ -1,22 +1,22 @@
 package com.wire.kalium.logic.feature.auth
 
-import com.wire.kalium.logic.feature.AuthenticatedDataSourceSet
+import com.wire.kalium.logic.di.UserStorage
 
 interface ClearUserDataUseCase {
```

Comment:
Maybe adding some documentations to this usecase, will be helpful for further reuse.

Reply:
We are missing at least a test for this, we should cover the clear method to reflect that we are not deleting `clientIds` in this step, or the other way around, to know what we are deleting 🙌